elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideResourceConfig.java

```diff
@@ -76,6 +84,25 @@ protected void configure() {
                 bind(elideSettings).to(ElideSettings.class);
                 bind(elideSettings.getDictionary()).to(EntityDictionary.class);
                 bind(elideSettings.getDataStore()).to(DataStore.class).named("elideDataStore");
+
+                // Binding async service
+                if(settings.enableAsync()) {
+                    AsyncQueryDAO asyncQueryDao = settings.getAsyncQueryDAO();
```

Comment:
Just create the DAO here.  We don't need a setting.  Users can override this function if they want something different.

Reply:
As discussed will leave this as is.